src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        
+        <div class="form-field">
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            minlength="2"
+            required
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
              minlength="2"
              required
            >
```
[checklist p.3 ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
pay attention to the code style and fix it all over the page